.github/workflows/ci.yml

```diff
@@ -65,3 +65,82 @@ jobs:
       - name: Upload to codecov
         run: |
           bash <(curl -s https://codecov.io/bash)
+
+  sentry:
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    steps:
+      # Checkout codebase
+      - name: Checkout snuba
+        uses: actions/checkout@v2
+
+      # Checkout Sentry and run integration tests against latest snuba
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+        with:
+          repository: getsentry/sentry
+          path: sentry
+
+      - name: Build snuba docker image
+        run: |
+          docker pull getsentry/snuba:latest || true
+          docker build . \
+            --build-arg PYTHON_VERSION=3.8 \
+            -t getsentry/snuba:${{ github.sha }} \
+            --cache-from getsentry/snuba:latest \
```

Comment:
Is this image built with `--build-arg BUILDKIT_INLINE_CACHE=1`? I read in the [documentation](https://docs.docker.com/engine/reference/commandline/build/#specifying-external-cache-sources) that it is needed in order to take advantage of `--cache-from`.

I don't see it in the source code:
https://github.com/getsentry/snuba/search?q=--build-arg+BUILDKIT_INLINE_CACHE%3D1

Reply:
It is not, we use Kaniko to build these images.